Enhanced_Zeus_Modules.sqf

```diff
@@ -14076,6 +14089,42 @@ MAZ_EZM_fnc_initFunction = {
 			["Vehicle repaired.","addItemOk"] call MAZ_EZM_fnc_systemMessage;
 		};
 
+		HYPER_EZM_fnc_setPlateNumber = {
+			params ["_entity"];
+			if (!(_entity isKindOf "Car_F")) exitWith {
+				["This is not a vehicle!","addItemFailed"] call MAZ_EZM_fnc_systemMessage;
+			};
+			[
+				"Set License Plate Number",
+				[ 
+					[
+						"EDIT",
+						"Plate Number (12 chars max)",
```

Comment:
Make this have a tooltip:
`["Plate Number", "The maximum length of a plate number is 15 characters"]`
Instead of `"Plate Number (12 chars max)"`